src/expression.js

```diff
@@ -73,6 +74,10 @@ var getKeyComputeData = function (key, scope, readOptions) {
 			return computeOrFunction();
 		}
 
+		if (computeOrFunction.computeData != null && computeOrFunction.hasOwnProperty("value")) {
```

Comment:
Is there a test for that scenario? I don't see that path getting hit when I run the tests.